service/domain/graph/builder_test.go

```diff
@@ -0,0 +1,307 @@
+package graph_test
```

Comment:
Everything here was moved from the other file. Only `TestSocialGraphBuilder_BanListExcludesContacts` is new.